package.json

```diff
@@ -12,7 +12,7 @@
     "fmt": "prettier --check '**/*.ts'",
     "fmt:write": "prettier --write '**/*.ts'",
     "prepare": "husky install",
-    "generate-types": "typechain --target=ethers-v5 --out-dir src/contracts './node_modules/@openzeppelin/contracts/build/contracts/ERC20.json'",
+    "generate-types": "typechain --target=ethers-v5 --out-dir src/contracts './node_modules/@openzeppelin/contracts/build/contracts/ERC20.json' './customabis/ERC721.json' './node_modules/@openzeppelin/contracts/build/contracts/ERC1155.json'",
```

Comment:
I see here we include the erc721 Abi directly in the project, but the other comes from a dependency. Could we also get the 722 from dependency?

Reply:
The problem is the way ethers.js handles overloaded functions in ABIs.
Here the ERC721 safeTransferFrom function is overloaded because the data parameter is optional.
So using the openzeppeling ERC721 ABI it would compile but throw an error during runtime, when hitting the submit button. 

I didn't know how to fix this, so I basically removed the overload of safeTransferFrom to fix this.

See this issue: https://github.com/ethers-io/ethers.js/issues/407

Reply:
Good enough. Could we get away with a smaller file here like that of `ERC721Interface`? In fact we might even be able to get away with only including the parts of the ABI for the functions we actually use (like transfer).